compiler/compile/OSRData.hpp.orig

```diff
@@ -0,0 +1,437 @@
+/*******************************************************************************
```

Comment:
I don't think you want to commit this...

Reply:
Thank you!  Removed in commit https://github.com/eclipse/omr/pull/5259/commits/bb10e07e5bb57e1b71d67a64f798c96554af378c